proptest/src/num/float_samplers.rs

```diff
@@ -0,0 +1,338 @@
+//-
+// Copyright 2022 Michael Wright
```

Comment:
The copyright should reflect the org, i.e. proptest-rs. Also, not sure why we have 2 licenses, MIT i think would be perfect on its own, but that's a larger discussion.

Reply:
technically our `CONTRIBUTING.md` say to use `either your name or "The proptest developers", at your choice.` We should update `CONTRIBUTING.md` if we only want "The proptest developers"

https://github.com/proptest-rs/proptest/blob/master/CONTRIBUTING.md#copyright-headers

Reply:
It seems simpler to just request `The proptest developers` then across the board. People will get the karma through git and github's existing features regardless.

I'll open a pr to update the readme 👍

Reply:
https://github.com/proptest-rs/proptest/pull/280